app/models/submission.rb

```diff
@@ -0,0 +1,18 @@
+class Submission < ApplicationRecord
+  SUBMISSION_STATUSES = [
+    "Received",
+    "In-progress",
+    "Declined",
+    "Accepted",
+    "Completed",
+    "Withdrawn",
+    "Editable",
+  ]
+  belongs_to :user
+  has_many :submission_packets, dependent: :destroy
+  has_many :poems, through: :submission_packets
+
+  validates_presence_of :title 
```

Comment:
It looks like `validates :attribute, presence: true` is used elsewhere in the app. Thoughts on using that here for consistency?